usr/lib/api/apiutil.c

```diff
@@ -58,7 +64,7 @@ CK_RV CreateProcLock(void)
         /* The slot mgr daemon should have already created lock,
          * so just open it so we can get a lock...
          */
-        xplfd = open(OCK_API_LOCK_FILE, O_RDONLY);
+        xplfd = open(OCK_API_LOCK_FILE, O_WRONLY);
```

Comment:
Here you also need to use the` OPEN_MODE` define from slotmgr.h.
It fails on Linux if this is `O_WRONLY`.